### ISSUE TITLE
Do not auto-register the plugin

### ISSUE THREAD

**Issue by weskerfoot** [AbelHeinsbroek/chartjs-plugin-crosshair#19]
https://github.com/AbelHeinsbroek/chartjs-plugin-crosshair/blob/fe674cfc91b4f4cf33bbbe666f0333bebbc84b1c/src/trace.js#L536

Could you not automatically register the plugin? It seems to be causing issues where the plugin gets enabled even if I try to specifically disable it by doing `plugins : { crosshair : false }` in my chart configuration.

**Comment by NagPraveen** [comment]
Yes this should be fixed. its getting auto registered on all charts.

As a temporary solution . i used like below for disable
```
options:{
...
plugins:{
crosshair:false
...
}
...
}
```
